src/iov.c

```diff
@@ -101,3 +101,51 @@ int ofi_truncate_iov(struct iovec *iov, size_t *iov_count, size_t trim_size)
 	}
 	return -FI_ETRUNC;
 }
+
+/* Copy 'len' bytes worth of src iovec to dst */
+int ofi_copy_iov_desc(struct ofi_iov_desc *src, size_t index, size_t offset,
+		      size_t len, struct ofi_iov_desc *dst)
```

Comment:
nit: The other iov copy routines have the destination as the first parameter, similar to memcpy.